src/4D_api.cpp

```diff
@@ -213,6 +213,19 @@ int pj_get_suggested_operation(PJ_CONTEXT *,
                                PJ_COORD coord)
 /**************************************************************************************/
 {
+    const auto normalizeLongitude = [](double x) {
+        if (x > 180.0) {
+            x -= 360.0;
+            if (x > 180.0)
+                x = fmod(x + 180.0, 360.0) - 180.0;
```

Comment:
Are you doing these `if` statements for performance reasons ("avoiding" to call `fmod`) or is there any other reason?

Reply:
yes, exactly for performance reasons